src/Containers/DropArea/DropArea.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { DropArea, IDropAreaProps } from './DropArea';
+
+export default {
+    title: 'Components/DropArea',
+    component: DropArea,
+    args: {},
+    argTypes: {
+        onDragLeave: { action: 'leaving drop are' },
+        onDragEnter: { action: 'entering drop area' },
+        onDropHandler: { action: 'files dropped' },
+        onClick: { action: 'you clicked!' },
+    },
+} as Meta;
+
+export const Basic: Story<IDropAreaProps> = (args) => <DropArea {...args} />;
+export const IsDragEnterTrue = Basic.bind({});
+IsDragEnterTrue.args = {
+    ...Basic.args,
+    isDragEnter: true,
+};
+export const OnClickPreventDefault = Basic.bind({});
+OnClickPreventDefault.args = {
+    ...Basic.args,
+    onClick: (e: React.MouseEvent) => {
```

Comment:
What is this doing here? Do you mean disabled state?

Reply:
this is to test if we can prevent default behaviour, and we can

Reply:
You should test disabled state not this. This is not a state.

Reply:
if we have an onClick prop, we must test it

Reply:
I have also tested the isDisabled state

Reply:
> if we have an onClick prop, we must test it

You test the onClick prop using action. 

And you test whether onClick is not emitting that action when in DISABLED state

Reply:
I have tested that, I wanted to test if I was able to prevent default behaviour. it's a test, it's not harmfull

Reply:
This is not a good test. Testing disabled state is better becuase it will test this behaviour as well. Remove this test, it's bloat.

Also style the disabled state to gray out the UI a bit